astrodb_utils/utils.py

```diff
@@ -115,113 +115,60 @@ def check_url_valid(url):
     return status
 
 
-def ingest_instrument(db, *, telescope=None, instrument=None, mode=None):
+def exit_function(msg, raise_error=True, return_value=None):
```

Comment:
Can we name this better? This function is not exiting, this is handling errors. Maybe a name like `handle_error`?

Reply:
In another note, we may want to have this as a decorator. This would allow us to add something like `@exit_function` above any function definition that we want to use this for without having to explicitly update the function at all.

May be worth exploring in the future.